statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1062,9 +1067,11 @@ def _fit_irls(self, start_params=None, maxiter=100, tol=1e-8,
         Fits a generalized linear model for a given family using
         iteratively reweighted least squares (IRLS).
         """
+        attach_wls = kwargs.pop('attach_wls', False)
```

Comment:
Is this kwarg documented anywhere?

Reply:
No, only in the description of this PR. It's currently a private backdoor, until we have a use for it outside of unit tests.

Reply:
I added it to the Notes section, mentioning it's internal